src/thunderbots/software/grsim_communication/grsim_command_primitive_visitor.cpp

```diff
@@ -124,7 +146,34 @@ void GrsimCommandPrimitiveVisitor::visit(
 
 void GrsimCommandPrimitiveVisitor::visit(const KickPrimitive &kick_primitive)
 {
-    // TODO: https://github.com/UBC-Thunderbots/Software/issues/93
+    Point kick_origin    = kick_primitive.getKickOrigin();
+    Angle kick_direction = kick_primitive.getKickDirection();
+
+    double final_speed_at_destination = 0.0;
+    Point perp_line_to_shot           = Point::createFromAngle(kick_direction.quarter());
+    Point normal_line_to_shot         = Point::createFromAngle(kick_direction.half());
```

Comment:
Same comment here as above regarding the use of `half()`.

I'm not entirely sure what this `Point` is supposed to be. Based on how you use it below, you should be able to just use the `kick_origin` instead (to make the line between the dest behind the kick origin, and the kick origin). So maybe this can be removed entirely?

Please correct me if you intended this to be something different.